Codecademy/gamut
# feat(Dialog): a new optional prop for the close button

### Overview


Added a new optional prop for the close button related to its tip.


### PR Checklist

- [x] Related to JIRA ticket: [ENT-233](https://skillsoftdev.atlassian.net/browse/ENT-233)

#### Testing Instructions

- Run `yarn start`
- Visit http://localhost:6006/?path=/docs/molecules-modals-dialog--dialog
- A new prop `closeButtonTipText` should be visible in the Code Playground section

#### PR Links and Envs

| Repository   | PR Link                                                  | PR Env                                                   |
| :----------- | :------------------------------------------------------- | :------------------------------------------------------- |
| LE Enterprise       | [LE Enterprise Link](https://github.com/codecademy-engineering/mono/pull/6987)  | [LE Enterprise Env](https://le-ent-pr-6987.dev-eks.codecademy.com/emodules/emod-javascript-requests-with-fetch-api)   |




[ENT-233]: https://skillsoftdev.atlassian.net/browse/ENT-233?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
